angular/src/components/settings/vault-timeout-input.component.ts

```diff
@@ -132,10 +136,18 @@ export class VaultTimeoutInputComponent implements ControlValueAccessor, Validat
       return { policyError: true };
     }
 
+    if (!this.getValidMinutesInput()) {
+      return { minTimeoutError: true };
+    }
+
     return null;
   }
 
   registerOnValidatorChange(fn: () => void): void {
     this.validatorChange = fn;
   }
+
+  getValidMinutesInput(): boolean {
+    return this.showCustom && this.customTimeInMinutes > 0;
```

Comment:
There's an error in this boolean logic - it'll return `false` (indicating an error) if a custom time out is not selected. 

You could go
```typescript
return !this.showCustom || this.customTimeInMinutes > 0
```

But I think breaking it out into a ternary if condition might be a clearer way of expressing it:

```suggestion
    return this.showCustom
      ? this.customTimeInMinutes > 0
      : true;
```

Reply:
I also liked that your previous implementation didn't have a magic number here. I recommend declaring a const outside the class to hold the minimum timeout value.

Reply:
I think the condition is clear